core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -397,6 +399,18 @@ public String toString() {
                           .toString();
     }
 
+    /**
+     * Reconfigure Server configuration.
```

Comment:
Could you add a description on
- `reconfigure()` should be called only after a `Server` has been started for the first time?
- Port newly added will not be applied.

Reply:
Done